src/main/java/com/redhat/devtools/intellij/tektoncd/actions/StartAction.java

```diff
@@ -97,12 +99,18 @@ public void actionPerformed(AnActionEvent anActionEvent, TreePath path, Object s
                     Map<String, Workspace> workspaces = model.getWorkspaces();
                     Map<String, String> inputResources = model.getInputResources();
                     Map<String, String> outputResources = model.getOutputResources();
+                    String runName = null;
                     if (element instanceof PipelineNode) {
-                        tkncli.startPipeline(namespace, element.getName(), params, inputResources, serviceAccount, taskServiceAccount, workspaces);
+                        runName = tkncli.startPipeline(namespace, element.getName(), params, inputResources, serviceAccount, taskServiceAccount, workspaces);
+
                     } else if (element instanceof TaskNode) {
-                        tkncli.startTask(namespace, element.getName(), params, inputResources, outputResources, serviceAccount, workspaces);
+                        runName = tkncli.startTask(namespace, element.getName(), params, inputResources, outputResources, serviceAccount, workspaces);
                     }
                     ((TektonTreeStructure)getTree(anActionEvent).getClientProperty(Constants.STRUCTURE_PROPERTY)).fireModified(element);
```

Comment:
I don't know why but if this line is put before the followLogsAction the tree is not refreshed and i don't see the new *run created below the task/pipeline. Could you move it below?

Reply:
Yes, sure. That strange...